context/configSchema.js

```diff
@@ -1,153 +1,121 @@
-const Schema = require('validate');
+const Validator = require('fastest-validator');
```

Comment:
sounds quick 😉

Reply:
Haha I know. The name is a bit lame but it was the only fit for purpose one I found.